modules/nf-core/hifiasm/tests/main.nf.test

```diff
@@ -0,0 +1,166 @@
+nextflow_process {
+
+    name "Test Process HifiAsm"
+    script "../main.nf"
+    process "HIFIASM"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "hifiasm"
+
+    test("Test HifiAsm - fq [] [] [] []") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id : 'test'],
+                    [ file(params.modules_testdata_base_path + 'genomics/homo_sapiens/pacbio/fastq/test_hifi.fastq.gz', checkIfExists: true)]
+                ]
+                input[1] = []
+                input[2] = []
+                input[3] = []
+                input[4] = []
+                """
+            }
+        }
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.raw_unitigs,
+                    process.out.processed_contigs,
+                    process.out.processed_unitigs,
+                    process.out.primary_contigs,
+                    process.out.alternate_contigs,
+                    process.out.paternal_contigs,
+                    process.out.maternal_contigs,
+                    process.out.versions
```

Comment:
are you leaving out any channel or could all out channels be captured with `process.out` ?

Reply:
I'm leaving out the log and the binary as those are not stable. The latter aren't documented as output files either but I figured some people might depend on them and I did not want to change the process definition.